release/7-3/ubi8/test-deps/meta.json

```
@@ -1,10 +1,10 @@
 {
     "IsLinux" : true,
     "UseLinuxVersion": false,
-    "osVersion": "Universal Base Image 8.4",
+    "osVersion": "Universal Base Image 8",
     "SkipGssNtlmSspTests": true,
     "shortTags": [
-        {"Tag": "8.4"}
+        {"Tag": "8"}
```

Should this be 8.6 to match base image ubi8.6 used on Line 4 of the the 7-3 ubi 8 dockerfile
```suggestion
        {"Tag": "8.6"}
```